### DISCUSSION TITLE
Trigger the reupload using terminal command

### DISCUSSION THREAD

**Discussion by arshavinel** [Natizyskunk/vscode-sftp#353] [Q&A] [answered]
Hello. I often use some terminal commands to update specific subfolders.

There is any command I can run from terminal so I can trigger the VSCode or the Extension so those folders/files are reuploaded?

**Comment by arshavinel** [comment]
@Natizyskunk, do you have any advice what I could do?

**Reply by Natizyskunk** [reply]
If I understand your question, you want to know if there's a way of calling an SFTP-extension command directly from your terminal to trigger file/folder ulpoad ? Am I correct ?

**Reply by arshavinel** [reply]
Hi, yes, that's what I need

**Reply by Natizyskunk** [reply, marked as answer]
Do you want to be able to run it inside the vscode terminal or from any terminal outside vscode ? Dou you also whant it to work even when the program is not runing ?

There's a little vscode cli program that can help for little things but AFAIK there's no such thing as running specific vscode and/or extensions operations directly from your terminal. It's not privded by microsoft and I don't think they will.

**Reply by arshavinel** [reply]
Running it inside the VSCode terminal is perfect and enough for my needs.

I don't need it to work even when the program is not running.
